docs/sources/flow/setup/install/windows.md

```diff
@@ -51,6 +51,11 @@ To do a silent install of Grafana Agent on Windows, perform the following steps.
 
    Replace `PATH_TO_INSTALLER` with the path where the unzipped installer executable is located.
 
+### Silent options
+
+* `/CONFIG=<path>` Path to config file. Default: `$INSTDIR\config.river`
+* `/DISABLEREPORTING=<yes|no>` Disable Analytics of Agent. Default: `no`
```

Comment:
```suggestion
### Silent install options

* `/CONFIG=<path>` Path to the configuration file. Default: `$INSTDIR\config.river`
* `/DISABLEREPORTING=<yes|no>` Disable [data collection][]. Default: `no`
```

We could add a link to the data collection topic. We need to add the following in the `doc/reference` shortcode block (somewhere between lines 72 and 77)

```
[data collection]: "/docs/agent/ -> /docs/agent/<AGENT_VERSION>/data-collection.md"
[data collection]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/monitor-infrastructure/agent/data-collection.md"
```